FD_frontend/src/containers/PostPage/ImageUpload.js

```diff
@@ -4,6 +4,7 @@ import { uploadImage } from '../../store/PostPage/actions'
 
 const mapStateToProps = (state) => {
   return {
+    statefunction: state
```

Comment:
We might later want to just pass the necessary props (or at least just the props corresponding to our page) for `mapStateToProp`. Passing the entire `props` might make components' code messy.